### PR TITLE
export input-buffer-vector and input-buffer-stream

### PR DESCRIPTION
**Pull request by mfiano**
It makes sense sometimes to use these.

### CODE REVIEW

File: src/package.lisp [hunk 1]
```diff
@@ -11,6 +11,7 @@
            #:buffer-position
 
            #:make-input-buffer #:input-buffer-stream
+           #:input-buffer-vector #:input-buffer-stream
```
Comment (rpav) [inline src/package.lisp]: I think `#:input-buffer-stream` is on the prior line?  But otherwise yeah.

**Comment by mfiano** [comment]
There.

**Comment by rpav** [comment]
Cool.  Thanks!

**Comment by mfiano** [comment]
No prob. Hope you are doing okay away from the usual place.

**Pull request merged**